README.md

```diff
@@ -116,12 +116,39 @@ Use this instead of `MVMT` in future steps.
 ```
 conda create -n MVMT python=3.9 -y
 conda activate MVMT
+```
+
+You could run into issues when installing dependencies. Check that your executable permissions are valid by running python and python3.9
+
+```
+python
+python3.9
+```
+
+If you get a permission denied error, find the location of the python executable inside of your environment that you want to change using the following command:
+
+```
+which python
+```
+
+Use the output and change the permissions using chmod.
 
-#If you haven't already, install [poetry](https://python-poetry.org/docs/). Now use this to install the majority of dependencies.
-#For some reason, poetry may not work if it's not in a graphical session. I think some form of authentication token is managed differently.
+```
+# What the command should look like: 
+sudo chmod ugo+rwx <CONDA ENV LOCATION>/bin/python3.9
```

Comment:
For the `chmod` command should we only limit it to execute permissions? I'm not sure if read and write permissions are necessary in this situation.

Reply:
Good catches. I think you're probably right that only exec is necessary. And yes, since the interpreters live on `/ofo-share`, we currently don't need `sudo`.